Article/Article.js

```diff
@@ -3,18 +3,31 @@
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement;
+    // STRETCH 3: Button to remove article marked as read
+    this.readButton = this.domElement.querySelector('.readButton');
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = this.domElement.querySelector('.expandButton');
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.textContent = 'Click to Expand';
+    // STRETCH 3: Click handler to remove article marked as read
+    this.readButton.addEventListener('click', this.readArticle.bind(this));
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.expandButton.addEventListener('click', this.expandArticle.bind(this));
   }
 
   expandArticle() {
     // Using our reference to the domElement, toggle a class to expand or hide the article.
-
+    this.domElement.classList.toggle('article-open');
+    if(this.expandButton.textContent === 'Click to Expand') {
+      this.expandButton.textContent = 'Click to Close';
+    } else {
+      this.expandButton.textContent = 'Click to Expand';
+    }
```

Comment:
remember that syntax i showed you other day, just another way of doing this without an if else statment
```
    this.expandButton.textContent = (this.expandButton.textContent === 'Click to Expand') ? 'Click to Close' : 'Click to Expand'
```
one liners are always nice to look at